python/GafferUI/ColorChooser.py

```diff
@@ -454,6 +451,121 @@ def __paintEvent( self, event ) :
 
 		self.__drawValue( painter )
 
+class _ColorFieldRowLayout( QtWidgets.QLayout ) :
+
+	def __init__( self ) :
+
+		QtWidgets.QLayout.__init__( self, None )
+
+		# No spacing between color field and sliders so the active component icons
+		# line up against the color field. Do add a little space between the sliders
+		# and the options button.
+		self.setSpacing( 0 )
+		self.__buttonSpacing = 4
```

Comment:
Super nitpicks inbound, but I do feel we've lost a little something in the clarity of the slider layout compared to what we have in current releases.

- We've lost our usual amount of spacing between the NumericWidget and the Slider.
- When the colour field is disabled, we still reserve a bunch of left padding in the component labels for colour field component indicators that will never be shown. This is mostly evident in the inline chooser, where we now indent the labels a fair amount compared to their positions in current Gaffer releases without the colour field.
- Ideally, the inline chooser would align the right edges of the sliders with the right edge of the colour swatch above them, and align the options button with the show/hide inline chooser button above.

I'm happy to punt on bullet point 3, but feel the others are worth considering.